### PR TITLE
[gamedig] fix import when using `moduleResolution: node16`

### PR DESCRIPTION
**Pull request by a-sync**
Not sure if i should add `moduleResolution: node16` to tsconfig.json to test for this?  

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
https://github.com/gamedig/node-gamedig/issues/569 
https://arethetypeswrong.github.io/?p=gamedig%405.0.0

### CODE REVIEW

**Comment by a-sync** [comment]
The `username` GamedigQuery option is only used by Palworld. I don't have a server to test it with but the implementation is here: https://github.com/gamedig/node-gamedig/pull/560/files#diff-a6a4a768aa273a6eb89750aa7e88c72bc3295ba9689d34da7618264a3ebbc6bfR7

**Comment by jakebailey** [comment]
> Not sure if i should add moduleResolution: node16 to tsconfig.json to test for this?

Yes, please do.

**Comment by a-sync** [comment]
getting the following error:
```
Error in gamedig
Error: 
	* /home/runner/work/DefinitelyTyped/DefinitelyTyped/types/gamedig/tsconfig.json:
	* Unexpected compiler option moduleResolution
```

@jakebailey please advise 🤷‍♂️

**Comment by jakebailey** [comment]
Set module, not moduleResolution

**Comment by a-sync** [comment]
Should i just add myself to the owners field? 🤔  
I don't see much activity from the others, and the types got an owerhaul anyway in v5. 🤷‍♂️

File: types/gamedig/tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "module": "commonjs",
+        "module": "node16",
```
Comment (MaximumWoahverdrive) [inline types/gamedig/tsconfig.json:3]: any reason for using node16 specifically?
I have no inherent issue with this, just curious.

**Comment by MaximumWoahverdrive** [comment]
@a-sync 
> Should i just add myself to the owners field?

Sure feel free, the meaning of the "owners" is documented [here](https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners) in the readme.

**Review by MaximumWoahverdrive — CHANGES_REQUESTED** [review]

File: types/gamedig/tsconfig.json (hunk 1 above)
Comment (a-sync) [reply types/gamedig/tsconfig.json:3]: the original issue was that the types didnt work with `moduleResolution: node16`: https://github.com/gamedig/node-gamedig/issues/569  

and the config was changed here to have a test for it:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69767#issuecomment-2179507649

**Review by MaximumWoahverdrive — APPROVED** [review]
Looks good to me

**Comment by a-sync** [comment]
Ready to merge

**Pull request merged**